catroid/src/org/catrobat/catroid/ui/fragment/FormulaEditorFragment.java

```diff
@@ -116,6 +115,7 @@ public void onCreate(Bundle savedInstanceState) {
 		currentBrickField = Brick.BrickField.valueOf(getArguments().getString(BRICKFIELD_BUNDLE_ARGUMENT));
 		cloneFormulaBrick(formulaBrick);
 		currentFormula = clonedFormulaBrick.getFormulaWithBrickField(currentBrickField);
+		setHasOptionsMenu(true);
```

Comment:
Please use a safer way to enable the options menu for devices that do not have such.
Example can be founf in #1222 or [here](https://github.com/MarkusHobisch/DroneTeam/blob/develop/catroid/src/org/catrobat/catroid/ui/fragment/FormulaEditorFragment.java#L111)